external/protobuf/README.md

```diff
@@ -3,7 +3,7 @@
 Protocol Buffers (a.k.a. `protobuf`) is Google's language-neutral, platform-neutral, extensible mechanism for serializing structured data. TizenRT features a port for `protobuf` version 3.5.1, with support for C++ language. 
 You can find details of `protobuf` at https://github.com/google/protobuf.
 
-The following sections detail the steps required to build `protobuf` on TizenRT, and to build and run TizenRT applications that use `protobuf`.
+The following sections detail the steps required to build `protobuf` on TizenRT, and to build and run TizenRT applications that use `protobuf`. In case you have just recently installed `gRPC`, and wish to install `Protobuf` to work along with `gRPC`, please ignore this document, and refer to the installation instructions under [Installing Protobuf with gRPC](https://github.com/Samsung/TizenRT/blob/master/external/grpc/README.md#pre-requisites) instead.
```

Comment:
This caution is too normal so that someone can miss this.
How about use **BOLD** character or make a *NOTE* tab or something else?

Reply:
Addressed your comment. Thanks for the suggestion.